### ISSUE TITLE
Required prop XXX was not specified in `BaseComponent`

### ISSUE THREAD

**Issue by sedubois** [relax/relax#324]
When opening http://localhost:8080/admin/templates/57ff981ac2b8dd657688fad2

`warning.js:45 Warning: Failed propType: Required prop `updateTemplate` was not specified in `BaseComponent`. Check the render method of `ConnectData`.`

`warning.js:45 Warning: Failed propType: Required prop `stylable` was not specified in `BaseComponent`. Check the render method of `Connect(BaseComponent)`.`

`warning.js:45 Warning: Failed propType: Required prop `eligable` was not specified in `BaseComponent`. Check the render method of `Connect(BaseComponent)`.`

**Comment by bruno12mota** [comment]
There are a lot of these indeed, they are usually easy fixes such as removing the required or fixing the type here and there. This is actually good and easy work for anyone who wants to contribute 😃

**Comment by sedubois** [comment]
Yes although it might be easier to first fix https://github.com/relax/relax/issues/328 😛

**Comment by searsaw** [comment]
this may be fixed in #414
